Gemfile

```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```

Comment:
I believe this was a Bundler issue. I will happily revisit re-adding it back in the future.